index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="icon" class="object-contain" href="../image/gsa_super_g-64.gif" type="image/icon type">
+<title>Google</title>
+
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link href="./dist/output.css" rel="stylesheet" />
+  </head>
+  <body class="relative flex flex-col gap-8 h-screen font-[Roboto] text-[0.3rem] md:text-base text-gray-600 w-full ">
```

Comment:
Kindly Check the different original website font sizes and work accordingly to them.